packages/theme/pages/Product.vue

```diff
@@ -136,14 +134,14 @@
                   </SfSelectOption>
                 </SfSelect>
               </template>
-              <template v-if="product.__typename === 'GroupedProduct'">
+              <template v-if="product !== null && product.__typename === 'GroupedProduct'">
```

Comment:
It's possible to avoid the condition duplication
```suggestion
          <template v-if="product !== null>
              <template v-if="product.__typename === 'GroupedProduct'">
              ...
              </template>
              <template v-else-if="product.__typename === 'someotherproductwhatever'">
              </template>
```

Reply:
The more nested templates can be removed aswell by just putting v-if directly on the grouped-product-selector

Reply:
ok